src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -499,6 +499,7 @@ export default class Aircraft {
         }
 
         if (this.model.weightclass === 'U') {
+
```

Comment:
could you remove this empty line?  I'm guessing this came over from a merge somewhere.